### ISSUE TITLE
Mobile display of New Post

### ISSUE THREAD

**Issue by piggybit** [ialbert/biostar-central#574]
Hello.
I just found that mobile display of New Post needs to be fixed.
Thanks and nice weekend!

![NewPost](https://user-images.githubusercontent.com/55419584/74589997-01b3a900-503d-11ea-82fa-6a4de2c9bb2f.jpg)

**Comment by piggybit** [comment]
hello @Natay would you mind checking this?
Thank you and nice weekend :).

**Comment by piggybit** [comment]
hello @Natay @ialbert 
Hope we can have it fixed.
Thank you very much!

**Comment by ialbert** [comment]
yes, we need to fix this, most likely we remove the formatting buttongs altogether

**Comment by Natay** [comment]
@piggybit , we will make user to fix this in the next release.
